src/main/java/seedu/address/logic/parser/ClientCommandParser.java

```diff
@@ -46,12 +60,44 @@ public ClientCommand parse(String flag, String arguments) throws ParseException
      * Parses the given {@code String} of arguments in the context of the AddCommand
      * and returns an AddCommand object for execution.
      *
-     * @param args string of arguments
+     * @param arguments string of arguments
      * @return an AddClientCommand object
      * @throws ParseException if the user input does not conform the expected format
      */
-    private AddClientCommand parseAddClientCommand(String args) throws ParseException {
-        return null;
+    private AddClientCommand parseAddClientCommand(String arguments) throws ParseException {
+        ArgumentMultimap argMultimap =
+                ArgumentTokenizer.tokenize(arguments, PREFIX_CLIENT_NAME, PREFIX_CLIENT_PHONE,
+                        PREFIX_CLIENT_EMAIL);
+
+        if (!arePrefixesPresent(argMultimap, PREFIX_CLIENT_NAME)
```

Comment:
Project ID should probably be compulsory as well, refer to documentation to confirm

Reply:
Done!